src/ingest_llm_as/parsers/python_ast_parser.py

```
@@ -543,71 +554,72 @@ def _create_module_element(self, source_code: str, file_path: Optional[str]) ->
-            tags={"module", "top-level"}
+            line_end=len(source_code.split("\n")),
+            tags={"module", "top-level"},
         )
-    
+
     def parse_directory(
-        self, 
-        directory_path: str, 
-        recursive: bool = True,
-        file_pattern: str = "*.py"
+        self, directory_path: str, recursive: bool = True, file_pattern: str = "*.py"
```

Ruff (0.13.3)

Unused method argument: `file_pattern`

(ARG002)